.idea/codeStyles/Project.xml

```diff
@@ -47,5 +47,10 @@
         <option name="TAB_SIZE" value="2" />
       </indentOptions>
     </codeStyleSettings>
+    <codeStyleSettings language="kotlin">
+      <indentOptions>
+        <option name="CONTINUATION_INDENT_SIZE" value="4" />
```

Comment:
Added this to a tracked code style as ktlint gets mad at me as I was using `8` continuation indent.